src/test/java/org/springframework/integration/dsl/test/jms/JmsTests.java

```diff
@@ -221,11 +249,38 @@ public IntegrationFlow jmsInboundFlow() {
 		}
 
 		@Bean
+		public IntegrationFlow pubSubFlow() {
+			return IntegrationFlows.from(Jms.publishSubscribeChannel(this.jmsConnectionFactory)
+											.destination("pubsub"))
+									.bridge(e -> e.id("bridgePubSub"))
```

Comment:
```
.from()
.channel()
```
A `BridgeHandler` is inserted automatically there.
From other side `.bridge(null)` is valid syntax, too.